Mathlib/Analysis/Convex/Polytope.lean

```diff
@@ -0,0 +1,58 @@
+/-
+Copyright (c) 2025 Yaël Dillies, Matthew Johnson. All rights reserved.
+Released under Apache 2.0 license as described in the file LICENSE.
+Authors: Yaël Dillies, Matthew Johnson
+-/
+import Mathlib.Analysis.Convex.Hull
+import Mathlib.Analysis.Convex.Combination
+import Mathlib.Algebra.Group.Pointwise.Finset.Basic
+
+/-!
+# Polytopes
+
+We define polytopes as convex hulls of finite sets.
+-/
+
+open scoped Pointwise
+
+variable {R 𝕜 E : Type*}
+
+section Semiring
+variable [Semiring R] [PartialOrder R] [IsOrderedRing R]
+variable [AddCommMonoid E] [Module R E] {s t : Set E} {x y : E}
+
+variable (R) in
+/-- A set is a polytope if it is the convex hull of finitely many points. -/
+def IsPolytope (s : Set E) : Prop := ∃ t : Finset E, s = convexHull R t
```

Comment:
```suggestion
def Set.IsPolytope (s : Set E) : Prop := ∃ t : Finset E, s = convexHull R t
```
for dot notation?

Reply:
I think that's actively harmful.
1. Other neighbooring concepts won't take a set as an argument, and therefore will find themselves in a different namespace.
2. In the future, this definition might move to the `Geometry` namespace, in which case all dot notation would have to be dropped again.
3. This isn't a property of sets, but of sets *in a vector space*